### PR TITLE
Feature/package

### PR DESCRIPTION
**Pull request by loriwitt**

### CODE REVIEW

File: opack/opack_construct.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+from construct import Adapter, Array, Computed, Filter, Float32l, Float64l, GreedyBytes, GreedyString, Int8ub, Int8ul, \
+    Int16ub, Int32ub, Int32ul, Int64ub, Int64ul, LazyBound, PascalString, Prefixed, RepeatUntil, Struct, Switch, \
+    Timestamp, this
+
+from opack.object_types import OBJ_TYPE_MAP, OPackObjectType, TerminatorObject
```
Comment (doronz88) [inline opack/opack_construct.py:7]: seems like only these objects are supposed to be the exported API from `object_types`. lets prefix all others with `_`

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+]
+dynamic = ["dependencies"]
+
+[project.optional-dependencies]
+test = ["pytest", "cmd2_ext_test"]
```
Comment (doronz88) [inline pyproject.toml]: remove the `cmd2_ext_test`

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: opack/opack_construct.py (hunk 1 above)
Comment (loriwitt) [reply opack/opack_construct.py:7]: fixed

File: pyproject.toml (hunk 2 above)
Comment (loriwitt) [reply pyproject.toml]: removed

**Review by doronz88 — APPROVED** [review]

**Pull request merged**